sklearn/utils/estimator_checks.py

```diff
@@ -1492,6 +1492,21 @@ def check_no_fit_attributes_set_in_init(name, Estimator):
                 'estimator but in the fit method. Attribute {!r} '
                 'was found in estimator {}'.format(attr, name))
 
+    if(Estimator.__name__ not in ["GaussianProcess", "RandomizedLasso",
```

Comment:
The description of the test `Check that Estimator.__init__ doesn't set trailing-_ attributes.` does not fit the second part of the test. Could you add a comment to describe the part you added?